### PR TITLE
Add eslint-plugin-vue

### PR DESCRIPTION
**Pull request by BennyAlex**

### CODE REVIEW

**Comment by KaelWD** [comment]
https://help.github.com/articles/closing-issues-using-keywords/

File: template/src/App.vue [hunk 1]
```diff
@@ -79,21 +79,21 @@
...
+        icon: 'bubble_chart',
+        title: 'Inspire'{{#if_eq lintConfig "airbnb"}},{{/if_eq}}
+      }],
+      miniVariant: false,
+      right: true,
+      rightDrawer: false,
+      title: 'Vuetify.js'{{#if_eq lintConfig "airbnb"}},{{/if_eq}}
+    }{{#if_eq lintConfig "airbnb"}};{{/if_eq}}
+  }{{#if_eq lintConfig "airbnb"}},{{/if_eq}}
+}{{#if_eq lintConfig "airbnb"}};{{/if_eq}}
```
Comment (KaelWD) [inline template/src/App.vue]: When vuejs/eslint-plugin-vue#349 is fixed, add
```js
"vue/script-indent": ["error", 2, {
  "baseIndent": 1,
  "switchCase": 0,
  "ignores": []
}]
```

**Comment by BennyAlex** [comment]
Closes #1

**Comment by maurop123** [comment]
I'm going to close this repo in favor of the new [webpack repo](https://github.com/vuetifyjs/webpack). You can check if this is still an issue in the new repo, and if so, feel free to reopen it there.

**Pull request closed without merging**